connectors/sources/gmail.py

```diff
@@ -100,6 +100,10 @@ def _message_doc(message):
     return es_doc
 
 
+def _filtering_enabled(filtering):
+    return filtering is not None and filtering.has_advanced_rules()
```

Comment:
What's the reasoning behind moving this to be a module-level func?

Reply:
It seems to be a "pythonic", that a method, which doesn't use the `self` reference can/should be either a static/class method (if it uses something from the class) or a module level/pure function. PyCharm also warns/hints at it.